README.md

```diff
@@ -1,2 +1,30 @@
-# opensourcery
- 
+#  Open Sourcery
+> A collection of templates for creating applications and websites.
+## Getting Started
+
+Make sure you have [node](https://nodejs.org/en/download/) installed. If you are not sure what version of node you are running or need help during installation please visit [here](https://nodejs.org/en/download/package-manager/). 
+
+## Setup
+> Update and install the following package first.
+``` bash
+$ npm install yarn
+```
+``` shell 
+$ git clone https://github.com/opensourceryclub/OpenSourcerySite.git
```

Comment:
This is incorrect. Contributors should for the repo (which sets their `origin` remote properly), make changes on said fork, and then submit a PR when they're done making changes.